xapian-bindings/perl/perl.i

```diff
@@ -405,6 +405,28 @@ sub add_rangeprocessor {
 }
 %}
 
+%feature("shadow") Xapian::QueryParser::add_prefix
+%{
+sub add_prefix {
+    my ($self, undef, $fproc) = @_;
+    if (defined($fproc) && defined(Scalar::Util::blessed $fproc)) {
+        push @{$self{_fproc}}, $fproc;
+    }
+    Xapianc::QueryParser_add_prefix( @_ );
+}
+%}
+
+%feature("shadow") Xapian::QueryParser::add_boolean_prefix
+%{
+sub add_boolean_prefix {
+    my ($self, undef, $fproc) = @_;
+    if (defined($fproc) && defined(Scalar::Util::blessed $fproc)) {
```

Comment:
CI is failing.  The build I looked was due to:

```
Can't locate object method "blessed" via package "Scalar::Util" at /home/travis/build/xapian/xapian/xapian-bindings/perl/Xapian.pm line 2638.
```

It seems `Scalar::Util::blessed` is not available in older Perl versions.

But I think we can avoid needing it and instead simply check `(ref($fproc) ne '')` here - we should either get passed a string or an object for that parameter, so we can assume a reference means an object - what happens with non-blessed references doesn't matter.  We shouldn't get passed an undefined value for the processor, but `(ref($fproc) ne '')` handles that suitably too.